test/index.js

```diff
@@ -1,7 +1,10 @@
 /* eslint-env mocha */
+import path from 'path';
+import process from 'process';
```

Comment:
The process import is not needed (and make the build fail on 0.12).
From the doc: `The process object is a global object and can be accessed from anywhere.`